src/ConnectClient/ConnectClient.php

```diff
@@ -2,11 +2,13 @@
 
 namespace Give\ConnectClient;
```

Comment:
I'd rather not have this as a domain in and of itself. @jonwaldstein where do you suggest this goes?

Reply:
@JasonTheAdams  @jonwaldstein Just a note, this class already existed before and is already used in other places with this domain, I just added the new methods and the local server support. However, I believe that makes more sense that it lives in the `PaymentGateways` folder.

Reply:
@glaubersilva Is this being used in any add-ons? Or is it safe to move?

Reply:
@JasonTheAdams In the addons that I have installed, I can't see any using it, just the core. But I don't have sure if any other addon is using it.

Reply:
@JasonTheAdams According to [this search](https://github.com/search?q=org%3Aimpress-org+ConnectClient&type=code), just a few places in the core are using it. Based on it, seems to be safe to change it. So, let me know if I can move forward with the change to this folder that I mentioned previously.

Reply:
Yeah, if we can move it then great. Still hoping for @jonwaldstein to weigh in on the location. I think `PaymentGateways\ConnectServer\ConnectClient` could be good. My only reservation on "PaymentGateway" is that this is also used for currency switcher. It's really anytime we need GiveWP to interact with external systems.

Reply:
Although its mainly used for gateways, the connect server is actually not specific to gateways.  It's sort of a general purpose api server.  That makes me think we should have a client in our framework domain. We can add to existing `Http` -> `Framework\Http\ConnectServer\Client`.  Then later we could even extract a new client for gateways specifically that uses the framework one _under-the-hood_.

Reply:
Yeah, I like that. Let's do that. 🙂

Reply:
Resolved in 5072d4b0b96b1ba151177e3a44f22a5bf15e9d4e